swarm/storage/dbstore.go

```diff
@@ -513,3 +541,37 @@ func (self *dbSyncIterator) Next() (key Key) {
 	self.it.Release()
 	return nil
 }
+
+func Import(sourcepath string, targetpath string) (uint64, error) {
+	sourcedb, err := NewDbStore(sourcepath, MakeHashFunc(defaultHash), defaultDbCapacity, defaultRadius)
+	if err != nil {
+		return 0, err
+	}
+	targetdb, err := NewDbStore(targetpath, MakeHashFunc(defaultHash), defaultDbCapacity, defaultRadius)
```

Comment:
is defaultDbCapacity here equal to our default of 20GB? because I'd want my targetDB to be bigger than that ( I set it to be bigger in the config.json, but ... )